add geometry.d.ts

nakakura opened the pull request:
Add Geometry Interfaces Spec

vvakame commented:
*geometry/geometry.d.ts*

check list

* [ ] is collect [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)?
  * https://www.npmjs.com/package/geometry
  * http://bower.io/search/?q=geometry
  * others?
* [X] has a [test file](http://definitelytyped.org/guides/contributing.html#tests)? (geometry/geometry-tests.ts or others)
* [ ] pass the Travic-CI test?

vvakame commented on npm-debug.log:
```diff
@@ -0,0 +1,36 @@
+0 info it worked if it ends with ok
```
please remove this file.

vvakame commented:
`geometry` is already in npm. but, I do not find a good name...

nakakura commented:
Thank you for your comment @vvakame san.
I changed the filename to "geometry-dom.d.ts" and removed npm-debug.log.
I searched npm and bower libraries.
There seemed to be no conflict.
I'm sorry to have troubled you

vvakame commented:
@nakakura さん。thanks mate!
